src/plugins/needs-info/index.js

```diff
@@ -20,9 +20,11 @@ Hi @${username}, thanks for the issue. It looks like there's not enough informat
 If you're reporting a bug, please be sure to include:
 
 1. The version of ESLint you are using (run \`eslint -v\`)
-2. What you did (the source code and ESLint configuration)
-3. The actual ESLint output complete with numbers
-4. What you expected to happen instead
+2. Is it a bug or enhancement ?
```

Comment:
I'm not convinced we should include this question. The issue in question could not fall under either of these categories and someone could potentially just answer "yes" to this question, requiring another communication cycle.

Reply:
I added this question in order to accommodate the situation when the issue template is not filled.